packages/std/src/deps.rs

```diff
@@ -0,0 +1,24 @@
+use crate::traits::{Api, Querier, Storage};
+
+/// Holds all external dependencies of the contract.
+/// Designed to allow easy dependency injection at runtime.
+/// This cannot be copied or cloned since it would behave differently
+/// for mock storages and a bridge storage in the VM.
+pub struct Deps<S: Storage, A: Api, Q: Querier> {
```

Comment:
Nice to pull this out. I was surprised to find it in traits when investigating

Reply:
Yeah this was probably the simplest way when the type was introduced to bundle the three traits